app/views/admin/merchants/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
+
+<h1>Admin Merchant Index Page</h1>
+
+<h2>Merchants</h2>
+
+<% @merchants.each do |merchant| %>
+  <section id= <%= "merchant-#{merchant.id}" %>>
+    <h3><%= merchant.name %></h3>
+    <p><%= "#{merchant.city}, #{merchant.state}" %></p>
+
+    <% if merchant.active? == true %>
```

Comment:
I think you could replace the conditional here with only `merchant.active?`, since that method already returns `true` or `false`.